docs/src/pages/layout/basics/basics.md

```diff
@@ -7,12 +7,11 @@
 [Responsive layouts](https://material.io/design/layout/responsive-layout-grid.html) in Material Design adapt to any possible screen size.
 We provide the following helpers to make the UI responsive:
 
-- [Grid](/layout/grid/):
-The grid creates visual consistency between layouts while allowing flexibility across a wide variety of designs.
+- [Grid](/layout/grid/): The grid creates visual consistency between layouts while allowing flexibility across a wide variety of designs.
+- [Breakpoints](/layout/breakpoints/): We provide a low-level API for using the breakpoints in a wide variery of context.
```

Comment:
We provide a low-level API that enables the use of breakpoints in a wide variety of contexts. ?